catalog/app/utils/AWS/S3.js

```diff
@@ -14,6 +14,7 @@ import * as Credentials from './Credentials'
 
 const DEFAULT_OPTS = {
   signatureVersion: 'v4',
+  s3UsEast1RegionalEndpoint: 'regional',
```

Comment:
What is this used for?

Reply:
to always use regional endpoints (instead of global `s3.amazonaws.com`), even for the default region (us-east-1)